it/controllers/studentLoans/UglAmountControllerISpec.scala

```diff
@@ -151,419 +91,164 @@ class UglAmountControllerISpec extends IntegrationTest with ViewHelpers with Emp
-
-          titleCheck(title, scenarioData.isWelsh)
-          h1Check(expectedH1)
-          captionCheck(expectedCaption)
-          textOnPageCheck(expectedParagraphCheckText, paragraphCheckSelector)
-          textOnPageCheck(hintText, hintTextSelector)
-          inputFieldValueCheck("amount", inputSelector, "84.73")
-
-          buttonCheck(expectedButtonText, continueButtonSelector)
-        }
-
-        "redirect to student loans cya page when there is no student loans data" in {
-
-          lazy val result = {
-            dropEmploymentDB()
-            authoriseAgentOrIndividual(scenarioData.isAgent)
-            insertCyaData(EmploymentUserData(
-                          sessionId,
-                          mtditid,
-                          nino,
-                          taxYearEOY,
-                          employmentId, isPriorSubmission = false, hasPriorBenefits = false, hasPriorStudentLoans = true,
-                          EmploymentCYAModel(
-                            EmploymentDetails(
-                              employerName = "Falador Knights",
-                              employerRef = Some("223/AB12399"),
-                              startDate = Some(startDate),
-                              didYouLeaveQuestion = Some(false),
-                              taxablePayToDate = Some(3000.00),
-                              totalTaxToDate = Some(300.00),
-                              currentDataIsHmrcHeld = false
-                            )
-                          )))
-            userDataStub(IncomeTaxUserData(), nino, taxYearEOY)
-
-            urlGet(url(taxYearEOY), follow = false, welsh = scenarioData.isWelsh,
-              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
-          }
-
-          result.status shouldBe SEE_OTHER
-          result.headers("Location").headOption shouldBe Some(controllers.studentLoans.routes.StudentLoansCYAController.show(taxYearEOY, employmentId).url)
+    "redirect to student loans cya page when there is no student loans data" in {
+
+      lazy val result = {
+        dropEmploymentDB()
+        authoriseAgentOrIndividual(false)
+        insertCyaData(EmploymentUserData(
+          sessionId,
+          mtditid,
+          nino,
+          taxYearEOY,
+          employmentId, isPriorSubmission = false, hasPriorBenefits = false, hasPriorStudentLoans = true,
+          EmploymentCYAModel(
+            EmploymentDetails(
+              employerName = "Falador Knights",
+              employerRef = Some("223/AB12399"),
+              startDate = Some("2022-04-01"),
+              didYouLeaveQuestion = Some(false),
+              taxablePayToDate = Some(3000.00),
+              totalTaxToDate = Some(300.00),
+              currentDataIsHmrcHeld = false
+            )
+          )))
+        userDataStub(IncomeTaxUserData(), nino, taxYearEOY)
 
-        }
+        urlGet(url(taxYearEOY), follow = false, welsh = false,
```

Comment:
The welsh content parameter boolean is defaulted to 'false' and so can be removed.